jgrapht-core/src/main/java/org/jgrapht/Graph.java

```diff
@@ -237,7 +230,8 @@ public interface Graph<V, E>
      *
      * @throws IllegalArgumentException if the graph supplier returns a vertex which is already in
      *         the graph
-     * @throws UnsupportedOperationException if the graph was not initialized with a vertex supplier
+     * @throws UnsupportedOperationException if this graph was not initialized with a vertex supplier
+     *                                       or if this graph disallows modification
```

Comment:
Since we're mentioning the "graph disallows modification" case here, I guess we should add the UnsupportedOperationException to all the graph modification methods in this interface, since any of them can throw that?

Reply:
Yeah, I guess we should.